src/chrome/content/rules/WebAssign.xml

```diff
@@ -4,6 +4,5 @@
   <target host="webassign.net" />
```

Comment:
Please also add:
```
api.webassign.net
feedback.webassign.net
webassignwired.webassign.net

Reply:
I've added the subdomains. Thanks!

Reply:
Sadly not all of them support HTTPS. Please use comments to note non-working hosts.

Reply:
Fixed, sorry about that.